### ISSUE TITLE
related_queries and related_topics not working (infinite loop)

### ISSUE THREAD

**Issue by bnorambuenay** [GeneralMills/pytrends#606]
Hi all, seems like pytrends broke again.

When trying to use functions **related_queries()** and/or **related_topics()**, execution enters in an infinite loop, without returning anything.

Below a screenshot after doing a Keyboard Interruption:
![Screenshot 2023-11-29 at 16 02 13](https://github.com/GeneralMills/pytrends/assets/65251793/90f0a35a-b871-40ee-b63e-6db410612f71)

Do you know what can be causing this error? How can we fix it?

**Comment by mobilityvalley** [comment]
My feeling is that the POST /explore request should post a payload (seems base64 encoded) but the python script does not, so Google detects the scraping

<img width="1159" alt="image" src="https://github.com/GeneralMills/pytrends/assets/303557/123a7025-ed02-414c-9b92-22eca68a0a1f">

I don't understand what the browser is  POSTing ....

**Comment by Helldez** [comment]
It's an invisible recaptcha

**Comment by thanhtoan1196** [comment]
any updates?

**Comment by nicktba** [comment]
Yep, now the data is returning a response with 0 in it, essentially rendering pytrends useless right now
